ocaml/idl/datamodel_pool.ml

```diff
@@ -1026,8 +1026,28 @@ let disable_repository_proxy =
 
 let set_uefi_certificates =
   call ~name:"set_uefi_certificates"
-    ~lifecycle:[(Published, "22.16.0", "")]
-    ~doc:"Sets the UEFI certificates for a pool and all its hosts"
+    ~lifecycle:
+      [
+        (Published, "22.16.0", "")
+      ; (Deprecated, "23.32.0", "use set_custom_uefi_certificates instead")
+      ]
+    ~doc:
+      "Set the UEFI certificates for a pool and all its hosts. Deprecated: use \
+       set_custom_uefi_certificates instead"
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; (String, "value", "The certificates to apply to the pool and its hosts")
+      ]
+    ~allowed_roles:_R_POOL_ADMIN ()
+
+let set_custom_uefi_certificates =
+  call ~name:"set_custom_uefi_certificates"
+    ~lifecycle:[(Published, "0.0.0", "")]
```

Comment:
Please leave this lifecycle empty, I believe it should work

Reply:
Done!